civis/tests/test_polling.py

```diff
@@ -97,6 +98,17 @@ def _check_result(self):
             polling_interval=0.1)
         pytest.raises(ZeroDivisionError, pollable.result, timeout=5)
 
+    def test_poll_on_creation(self):
+        poller = mock.Mock(side_effect=Response({"state": "running"}))
+        pollable = PollableResult(poller,
+                                  (),
+                                  polling_interval=0.1,
+                                  poll_on_creation=False)
+        repr(pollable)
+        assert poller.call_count == 0
+        time.sleep(0.2)
```

Comment:
Will this test still work if the polling interval is 0.01 and the sleep is 0.02? Adding 0.2 seconds here and there in the tests will add up.